projects/admin/src/app/record/detail-view/circ-policy-detail-view/circ-policy-detail-view.component.ts

```diff
@@ -53,7 +53,7 @@ export class CircPolicyDetailViewComponent implements OnInit, OnDestroy {
 
   /** checkout is allowed ? */
   get checkoutIsAllowed() {
-    return this.record && this.record.metadata.checkout_duration > 0;
+    return this.record && this.record.metadata.hasOwnProperty('checkout_duration');
```

Comment:
Can be `record.metadata['checkout_duration']`?

Reply:
not sure about that. If the `checkout_duration` is equal to 0, the checkout is allowed only for the current day. But with your proposal, javascript will evaluate 0 to `false` and return than checkout isn't allowed.
If the `checkout_duration` property isn't set, the checkout is disable ; if it exists with any value, the checkout is allowed.